### PR TITLE
Fulfill license obligations for helm generated files

### PR DESCRIPTION
**Pull request by strieflin**
Signed-off-by: Sven Trieflinger <sven.trieflinger@de.bosch.com>

### CODE REVIEW

**Review by sbckr — APPROVED** [review]
:+1:

File: castor-service/charts/castor/templates/_helpers.tpl [hunk 1]
```diff
@@ -1,4 +1,8 @@
 {{/* vim: set filetype=mustache: */}}
+{{/* Copyright (c) 2021 - for information on the respective copyright owner */}}
+{{/* see the NOTICE file and/or the repository https://github.com/carbynestack/ephemeral. */}}
```
Comment (sbckr) [inline castor-service/charts/castor/templates/_helpers.tpl:3]: wrong repository link -> https://github.com/carbynestack/castor

**Review by sbckr — CHANGES_REQUESTED** [review]
See file comment

File: castor-service/charts/castor/templates/_helpers.tpl (hunk 1 above)
Comment (strieflin) [reply castor-service/charts/castor/templates/_helpers.tpl]: fixed

**Review by sbckr — APPROVED** [review]
:+1:

**Pull request merged**